README.md

```diff
@@ -35,12 +35,6 @@ This library also includes tools for some common uses of weak and soft reference
 
 Ruby does come with the `WeakRef` class in the standard library. However, there are [issues with this class](https://bugs.ruby-lang.org/issues/4168) across several different Ruby runtimes. This gem provides a common interface to weak references that works across MRI, Ruby Enterprise Edition, YARV, JRuby and Rubinius.
 
-1. MRI and REE 1.8 - `WeakRef` extends from Delegator which is a very heavy weight class under Ruby 1.8. Creating a `WeakRef` object will allocate thousands of other objects and use up hundreds of kilobytes of memory. This makes `WeakRef` all but unusable even if you only need several hundred of them.
-2. YARV 1.9 - `WeakRef` is unsafe to use because the garbage collector can run in a different system thread than a thread allocating memory. This exposes a bug where a `WeakRef` may end up pointing to a completely different object than it originally referenced.
```

Comment:
Support for MRI 1.9 has ended, but I've never seen any data indicating that people aren't still using it in production. So I would hesitate to say it's actually dead. For this gem and this release I think dropping 1.9 support is OK. We aren't adding any new features or fixing bugs. The main goal of this release is to drop support for old versions of Ruby (mainly MRI 1.8 and IronRuby). MRI 1.9 users will still be able to use the 1.x release so we won't be disenfranchising any users. I would argue against dropping support for MRI 1.9 in any of our other gems, however.